pkg/hs/urbit-king/lib/Urbit/Vere/Eyre/Serv.hs

```diff
@@ -140,8 +138,12 @@ openFreePort hos = do
       Right ps -> pure (Right ps)
  where
   doBind sok = do
-    adr <- Net.inet_addr hos
-    Net.bind sok (Net.SockAddrInet Net.defaultPort adr)
+    adr <-
+      Net.getAddrInfo (Just Net.defaultHints) (Just hos) Nothing >>= \case
+        [] -> error ("unable to determine numeric hostname from " ++ hos)
+        x:_ -> pure (Net.addrAddress x)
```

Comment:
This will probably work, but did you manually test that it does?

Reply:
In the process of going through various tests on Linux, hold off from any kind of merge until verified.